### PR TITLE
Capsule design team output [HTTP/3]

### PR DESCRIPTION
**Pull request by ekinnear**
(Initial pull request for discussion purposes)

### CODE REVIEW

**Review by martinthomson — APPROVED** [review]
Two bytes for a close seems like a good choice.

File: draft-ietf-webtrans-http3.md [hunk 1]
```diff
@@ -166,18 +172,90 @@ The `webtransport` HTTP Upgrade Token uses the Capsule Protocol as defined in
...
+
+An HTTP capsule {{HTTP-DATAGRAM}} called WT_MAX_STREAMS is introduced to inform
+the peer of the cumulative number of streams of a given type it is permitted to
+open.  A WT_MAX_STREAMS capsule with a type of 0x12 applies to bidirectional
+streams, and a WT_MAX_STREAMS capsule with a type of 0x13 applies to
+unidirectional streams.
+
+~~~
+WT_MAX_STREAMS Capsule {
+  Type (i) = 0x12..0x13,
```
Comment (DavidSchinazi) [inline draft-ietf-webtrans-http3.md]: Similarly, let's use capsule types in the 4-byte encoding while we're still iterating

**Review by DavidSchinazi — APPROVED** [review]

**Comment by ekinnear** [comment]
Using https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-http3/issues/77 to track introducing a capsule the reliably communicates streams and their association to a WT session, splitting that out from here.

**Comment by ekinnear** [comment]
And opened https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-http3/pull/86 to match, thanks for the reviews and comments!

**Pull request closed without merging**